grape/main.rb

```diff
@@ -42,17 +10,28 @@ class API < Grape::API
 
     resource :tasks do
       get "/" do
-        future_tasks
+        to_json(Tasks.futures)
+        #Tasks.futures
       end
     end
 
     helpers do
-      def future_tasks
-        TASKS.select {|task| future_task?(task)}
-      end
+      def to_json tasks
```

Comment:
You may need to check out the documentation again, grape already has a way of dealing with this: https://github.com/ruby-grape/grape#restful-model-representations